setup.py

```diff
@@ -58,7 +58,7 @@
     requirements.append('importlib')
 
 setup(name='pybloomfiltermmap',
-      version="0.4.0j",
+      version="0.4.1j",
```

Comment:
One comment, though, that I have just remembered. Newer versions of `pip` do not like this versioning scheme and will refuse to build wheels because they do not comply with PEP 440. We should be using `0.4.1+j`.